odoo/tools/sql.py

```diff
@@ -217,7 +217,7 @@ def fix_foreign_key(cr, tablename1, columnname1, tablename2, columnname2, ondele
     if not found:
         return add_foreign_key(cr, tablename1, columnname1, tablename2, columnname2, ondelete)
 
-def has_pg_trgm(cr):
+def install_pg_trgm(cr):
     cr.execute("select installed_version from pg_available_extensions where name='pg_trgm'")
```

Comment:
Can you change this to be more readable (with uppercase from sql keyword) - (same for the next one ? )